packages/speech/src/index.js

```diff
@@ -0,0 +1,919 @@
+ * @param {string} recognitionConfig.encoding - Encoding of audio data
+ *     sent in all `RecognitionAudio` messages. See
+ *     [`RecognitionConfig.encoding`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ * @param {number} recognitionConfig.sampleRate - Sample rate in Hertz
+ *     of the audio data sent in all RecognitionAudio messages. See
+ *     [`RecognitionConfig.sampleRate`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ * @param {string=} recognitionConfig.languageCode - See
+ *     [`RecognitionConfig.languageCode`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `"en-US"`.
+ * @param {number=} recognitionConfig.maxAlternatives - See
+ *     [`RecognitionConfig.maxAlternatives`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `1`.
+ * @param {boolean=} recognitionConfig.profanityFilter - See
+ *     [`RecognitionConfig.profanityFilter`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `false`.
+ * @param {string[]=} recognitionConfig.speechContext - See
+ *     [`RecognitionConfig.speechContext`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `[]`.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object[]} callback.syncRecognizeResponse -
+ *     `SyncRecognizeResponse` objects. See
+ *     [`SyncRecognizeResponse`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#syncrecognizeresponse).
+ * @param {object} callback.apiResponse - Raw API response. See
+ *     [`SyncRecognizeResponse`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#syncrecognizeresponse).
+ *
+ * @example
+ * //-
+ * // <h3>Run speech detection over a local file</h3>
+ * //-
+ * speech.detect('./bridge.raw', {
+ *   encoding: 'LINEAR16',
+ *   sampleRate: 16000
+ * }, function(err, result, apiResponse) {
+ *   //   result = {
+ *   //     results: [
+ *   //       {
+ *   //         alternatives: [
+ *   //           {
+ *   //             transcript: "...",
+ *   //             confidence: 0.88
+ *   //           }
+ *   //         ]
+ *   //       }
+ *   //     ]
+ *   //   }
+ * });
+ *
+ * //-
+ * // <h3>Run speech detection over a remote file</h3>
+ * //
+ * // *Note: This is not an officially supported feature of the Speech API. Our
+ * // library will make a request to the URL given, and send that upstream.*
```

Comment:
How about *`google-cloud` will make a request to the URL given and send the file contents to the upstream API*?